packages/web-components/src/radio-group/README.md

```diff
@@ -0,0 +1,127 @@
+# Radio Group
+
+> RadioGroup lets people select a single option from two or more Radio items. Use RadioGroup to present all available choices if there's enough space..
+
+<br />
+
+## **Design Spec**
+
+[Link to Radio Design Spec in Figma](https://www.figma.com/file/4XWsJrlpEcuEpUnZbtoIBU/Radio?node-id=1295%3A0&t=2wXnjT4ybxIxT6wu-0)
+
+<br />
+
+## **Engineering Spec**
+
+As defined by the [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/radio/)..
+
+> A radio group is a set of checkable buttons, known as radio buttons, where no more than one of the buttons can be checked at a time. Some implementations may initialize the set with all buttons in the unchecked state in order to force the user to check one of the buttons before moving past a certain point in the workflow.
+
+### Use Case
+
+Used anywhere an author might group a list of radio options.
+
+<br />
+
+## Class: `RadioGroup`
+
+<br />
+
+### **Component Name**
+
+<br />
+
+`fluent-radio-group`
+
+<br />
+
+### **Fields**
+
+| Name          | Privacy | Type                     | Default      | Description                                                                                           |
+| ------------- | ------- | ------------------------ | ------------ | ----------------------------------------------------------------------------------------------------- |
+| `disabled`    | public  | `boolean`                | `false`      | Disables the radio group and child radios.                                                            |
+| `name`        | public  | `string`                 |              | The name of the radio group. Setting this value will set the name value for all child radio elements. |
+| `value`       | public  | `string`                 |              | The value of the checked radio.                                                                       |
+| `orientation` | public  | `horizontal \| vertical` | `horizontal` | The orientation of the group                                                                          |
+| default slot  | public  | `HTMLElement[]`          |              | The default slot expecting Radio items                                                                |
+
+<br />
+
+### **Methods**
+
+| Name           | Privacy   | Description | Parameters | Return | Inherited From |
+| -------------- | --------- | ----------- | ---------- | ------ | -------------- |
+| `nameChanged`  | protected |             |            | `void` |                |
+| `valueChanged` | protected |             |            | `void` |                |
```

Comment:
just a note that while not enumerated, each attr includes a `[attribute]Changed()` callback 👍